Refactored SQL Observability DB/Cloud console pages to be shared. (#12263)
cockroachdb/docs: _includes/common/ui/sessions-page.md

{% if page.cloud == true %}
  {% capture link_prefix %}../{{site.versions["stable"]}}/{% endcapture %}
  {% assign page_prefix = "" %}
{% else %}
  {% assign link_prefix = "" %}
  {% assign page_prefix = "ui-" %}
{% endif %}

## Sessions list

Use the **Sessions** list to see the open sessions in the cluster. This includes active and idle sessions.

{{site.data.alerts.callout_info}}
A session is *active* if it has an open transaction (including implicit transactions, which are individual SQL statements), and *idle* if it has no open transaction. Active sessions consume hardware resources.
{{site.data.alerts.end}}

- If a session is active, the most recent SQL statement is displayed in the **Statement** column.
- If a session is idle, **Transaction Duration**, **Statement Duration**, and **Statement** will display `N/A`.

> - To view [details of a session](#session-details), click the **Session Duration**.

Reviewer: ```suggestion
- To view [details of a session](#session-details), click **Session Duration**.
```
```suggestion
- To view [details of a session](#session-details), click the **Session Duration**.
```

{{site.data.alerts.callout_info}}
An active session can have an open transaction that is not currently running SQL. In this case, the **Statement** and **Statement Duration** columns will display `N/A` and **Transaction Duration** will display a value. Transactions that are held open can cause [contention]({{ link_prefix }}performance-best-practices-overview.html#understanding-and-avoiding-transaction-contention).
{{site.data.alerts.end}}

{% if page.cloud != true %}

> <img src="{{ 'images/v21.2/ui-sessions-page.png' | relative_url }}" alt="DB Console Database Tables View" style="border:1px solid #eee;max-width:100%" />

Reviewer: If we have Alt text it should be accurate. Right now it is the same in all screenshots.

## Session details

Click the **Session Duration** of any session to display details and possible actions for that session.

{% if page.cloud != true %}
<img src="{{ 'images/v21.2/ui-sessions-details-page.png' | relative_url }}" alt="DB Console Database Tables View" style="border:1px solid #eee;max-width:100%" />
{% endif %}

- **Session** shows the ID of the connected session.
	- **Session Start Time** shows the timestamp at which the session started.
	- **Gateway Node** <a name="session-details-gateway-node"></a> shows the node ID and IP address/port of the [gateway]({{ link_prefix }}architecture/life-of-a-distributed-transaction.html#gateway) node handling the client connection.
	- **Client Address** shows the IP address/port of the client that opened the session.
	- **Memory Usage** shows the amount of memory currently allocated to this session, followed by the maximum amount of memory this session has ever allocated.
	- The **Terminate Session** button ends the session. The client that holds this session will receive a "connection terminated" event.
	- The **Terminate Statement** button ends the SQL statement. The session running this statement will receive an error.

> - **Transaction** will display the following information for an open transaction.

Reviewer: ```suggestion
- **Transaction** displays the following information for an open transaction.
```
```suggestion
- **Transaction** will display the following information for an open transaction.
```

	- **Transaction Start Time** shows the timestamp at which the transaction started.
	- **Number of Statements Executed** shows the total number of SQL statements executed by the transaction.
	- **Number of Retries** shows the total number of [retries]({{ link_prefix }}transactions.html#transaction-retries) for the transaction.
	- **Number of Automatic Retries** shows the total number of [automatic retries]({{ link_prefix }}transactions.html#automatic-retries) run by CockroachDB for the transaction.
	- **Priority** shows the [priority]({{ link_prefix }}transactions.html#transaction-priorities) for the transaction.
	- **Read Only?** shows whether the transaction is read-only.
	- **AS OF SYSTEM TIME?** shows whether the transaction uses [`AS OF SYSTEM TIME`]({{ link_prefix }}performance-best-practices-overview.html#use-as-of-system-time-to-decrease-conflicts-with-long-running-queries) to return historical data.
	- **Memory Usage** shows the amount of memory currently allocated to this transaction, followed by the maximum amount of memory this transaction has ever allocated.

> - **Statement** will display the following information for an active statement.

Reviewer: ```suggestion
- **Statement** displays the following information for an active statement.
```
```suggestion
- **Statement** will display the following information for an active statement.
```

> 	- The SQL statement is shown.

Reviewer: ```suggestion
	- The SQL statement.
```
```suggestion
	- The SQL statement is shown.
```

> 	- **Execution Start Time** shows the timestamp at which the statement was run.

Reviewer: ```suggestion
	- **Execution Start Time** the timestamp at which the statement was run.
```
```suggestion
	- **Execution Start Time** shows the timestamp at which the statement was run.
```

> 	- **Distributed Execution?** shows whether the statement uses [Distributed SQL (DistSQL)]({{ link_prefix }}architecture/sql-layer.html#distsql) optimization.

Reviewer: ```suggestion
	- **Distributed Execution?** whether the statement uses [Distributed SQL (DistSQL)]({{ link_prefix }}architecture/sql-layer.html#distsql) optimization.
```
```suggestion
	- **Distributed Execution?** shows whether the statement uses [Distributed SQL (DistSQL)]({{ link_prefix }}architecture/sql-layer.html#distsql) optimization.
```
